packages/fxa-settings/src/pages/ResetPassword/CompleteResetPassword/index.tsx

```diff
@@ -231,45 +221,14 @@ const CompleteResetPassword = ({
           case IntegrationType.OAuth:
             // allows a navigation to a "complete" screen or TOTP screen if it is setup
             // TODO: check if relier has state
```

Comment:
Probably want to remove these two comments.

Also should we just remove what's in `case IntegrationType.OAuth` check?